src/Traveler.js

```diff
@@ -0,0 +1,23 @@
+import AllTravelers from "./AllTravelers";
+
+class Traveler extends AllTravelers {
```

Comment:
I'm not exactly sure why you are extending this Traveler from AllTravelers? I see that the traveler has an Id property but you should have access to that Id in the AllTravelers class as well. Should a traveler have access to all of the methods that AllTravelers has?

Reply:
My AllTravelers class gets all the data I need to display on the Agent side and then my Traveler side extends that because it needs all the same methods but just for a single user because if you login as a single user, you don't need to see everyone's info like the Agent does.

Reply:
Okay that makes sense, kinda for privacy reasons